[2025-01-01T03:42:47.739+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-01T03:42:47.760+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp.my_name scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-01T03:42:47.766+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp.my_name scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-01T03:42:47.767+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-01-01T03:42:47.776+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): my_name> on 2024-12-31 00:00:00+00:00
[2025-01-01T03:42:47.781+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=292) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-01T03:42:47.782+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scheduler_dag_with_cron_exp', 'my_name', 'scheduled__2024-12-31T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/scheduler_dag_with_cron_exp.py', '--cfg-path', '/tmp/tmpmsqxsu43']
[2025-01-01T03:42:47.783+0000] {standard_task_runner.py:72} INFO - Started process 293 to run task
[2025-01-01T03:42:47.783+0000] {standard_task_runner.py:105} INFO - Job 55: Subtask my_name
[2025-01-01T03:42:47.821+0000] {task_command.py:467} INFO - Running <TaskInstance: scheduler_dag_with_cron_exp.my_name scheduled__2024-12-31T00:00:00+00:00 [running]> on host b0ef9544b3b5
[2025-01-01T03:42:47.873+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tanzid' AIRFLOW_CTX_DAG_ID='scheduler_dag_with_cron_exp' AIRFLOW_CTX_TASK_ID='my_name' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-31T00:00:00+00:00'
[2025-01-01T03:42:47.873+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-01T03:42:47.900+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-01T03:42:47.910+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo My Name is Md. Tanzid Hasan']
[2025-01-01T03:42:47.925+0000] {subprocess.py:99} INFO - Output:
[2025-01-01T03:42:47.950+0000] {subprocess.py:106} INFO - My Name is Md. Tanzid Hasan
[2025-01-01T03:42:47.951+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-01T03:42:47.977+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-01T03:42:47.977+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scheduler_dag_with_cron_exp, task_id=my_name, run_id=scheduled__2024-12-31T00:00:00+00:00, execution_date=20241231T000000, start_date=20250101T034247, end_date=20250101T034247
[2025-01-01T03:42:48.000+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-01T03:42:48.011+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-01T03:42:48.012+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
